content/source/docs/enterprise/index.html.md

```diff
@@ -17,19 +17,14 @@ For independent teams and small to medium-sized businesses, Terraform Cloud is a
 
 ### Note About Product Names
 
-Before mid-2019, Terraform Enterprise used to be called Private Terraform Enterprise.
-
-We're still updating names throughout their UI, documentation, and supporting tools. While we finish these updates, you might still see occasional references to their old names:
-
-Component               | New name             | Old name
-------------------------|----------------------|----------------------
-Primary application     | Terraform Cloud      | Terraform Enterprise
-On-premise distribution | Terraform Enterprise | Private Terraform Enterprise (PTFE)
+Before mid-2019, all distributions of Terraform Cloud used to be called Terraform Enterprise; the self-hosted distribution was called Private Terraform Enterprise (PTFE). These previous names sometimes still appear in supporting tools (like [the `tfe` Terraform provider](https://registry.terraform.io/providers/hashicorp/tfe/latest), which is also intended for use with Terraform Cloud).
 
 ## Who is This Documentation For?
 
-The Terraform Enterprise documentation is for admins and operators who install and maintain their organization's Terraform Enterprise instance. It is not intended for normal users of the Terraform Cloud application.
+The **Deployment and Operation** section of the Terraform Enterprise documentation is for admins and operators who install and maintain their organization's Terraform Enterprise instance. It is not intended for end users of the Terraform Enterprise application.
+
+The **Application Usage** section of the Terraform Enterprise documentation is the same content as the [Terraform Cloud documentation](/docs/cloud/index.html). It is intended for everyone who uses the Terraform Cloud application to provision and manage infrastructure; this includes Terraform Enterprise users, since Terraform Enterprise and Terraform Cloud are the same application.
```

Comment:
```suggestion
The **Application Usage** section of the Terraform Enterprise documentation is intended for application end users. It is the same content as the [Terraform Cloud documentation](/docs/cloud/index.html) because the application is shared between both products. 
```